caffe2/python/workspace.py

```diff
@@ -82,6 +97,12 @@ def _GetFreeFlaskPort():
         # rather than 24x7 service.
         return port
 
+def GpuDeviceType():
```

Comment:
Since has_hip_support and hip_gpu_support is not going to be changed once initialized, I think GpuDeviceType should be a variable instead of function.

Reply:
Makes sense. I will make that change.